e2e/nomostest/artifactregistry/image.go

```diff
@@ -51,6 +50,12 @@ func SetupImage(nt *nomostest.NT, name, version string) (*Image, error) {
 	if version == "" {
 		return nil, fmt.Errorf("image version must not be empty")
 	}
+
+	// Version will error out if longer than 20 characters
+	v, err := validateImageVersion(version)
```

Comment:
I don't think this needs to return the version if it's not being changed. Just validate and return the error.

Reply:
removed